packages/drivers/odsp-driver/src/odspDocumentService.ts

```diff
@@ -88,6 +89,11 @@ function writeLocalStorage(key: string, value: string) {
 export class OdspDocumentService implements IDocumentService {
     protected updateUsageOpFrequency = startingUpdateUsageOpFrequency;
 
+    readonly policies = {
+        // Disable prefetching when using ODSP by default.
+        caching: ServiceCachingPolicy.None,
```

Comment:
Should this be renamed to "full" or something similar? I.e. service (driver) caching policy is not none, it's actually the opposite.

Reply:
It's actually Prefetch below. I think here and in Container.ts we should use Prefetch, not none.

Reply:
Better? (Also a it confused by your second comment - we are *not* prefetching in ODSP, correct?)

Reply:
The problem I see is that comment contradicts code:

```
    readonly policies = {
        // Disable prefetching when using ODSP by default.
        caching: ServiceCachingPolicy.NoCaching,
    };
```
Comments says - disable prefetching in container, because ODSP does some caching.
I read next line as "service caching policy is no caching". But comment (and behavior) contradicts it - service caching policy is actually - cache everything!

So either ServiceCachingPolicy should be renamed to LoaderCachingPolicy (then it's instruction from ODSP - "container please do not cache"), or it has to be ServiceCachingPolicy.Full or equivalent that provides info to loader "driver is doing caching, please take it into account"

Reply:
Ah ok, thanks for the clarification! I prefer renaming to LoaderCaching for clarity.

Reply:
It's just wording, and how people will read it :) The code is correct, but someone reading it wrong will make it wrong, it's just matter of time :)